pvlib/tests/test_pvsystem.py

```diff
@@ -174,7 +174,7 @@ def test_retrieve_sam_cecinverter():
     ]
     inverter = 'Yaskawa_Solectria_Solar__PVI_5300_208__208V_'
     assert inverter in data
-    assert set(data[inverter].keys()) == set(keys)
+    assert set(data[inverter]) == set(keys)
```

Comment:
For the two cases in this file, we are actually dealing with pandas Series, not dicts, so the semantics are different: `set(Series)` returns a set of the values instead of the keys.  So these two lines should keep the `.keys()` (or be changed to use `assert_index_equal` or something)`.